Jenkinsfile

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env groovy
```

Comment:
You *may* remove the shebang, assuming your only interest was in GitHub syntax coloring. If you were relying on this to trigger Groovy support in other editors, then this or (more simply)

```groovy
#!groovy
```

would need to remain unless and until those edits start treating `Jenkinsfile` as a recognized filename.